spec/string_foundation/symbol_spec.rb

```diff
@@ -0,0 +1,48 @@
+# ==============================================================================
+# SPEC - STRING FOUNDATION - SYMBOL
+# ==============================================================================
+# frozen_string_literal: true
+describe "[ Symbol Methods ]" do
+  let(:string) { "string" }
+  let(:expected_symbol) { :string }
+
+  # ----------------------------------------------------------------------------
+  # Check Whether Character Is A Specific Symbol
+  # ----------------------------------------------------------------------------
+  describe "CHECK WHETHER CHARACTER IS A SPECIFIC SYMBOL ::" do
+    subject { string.is_sym?(expected_symbol) }
+
+    context "when an argument is not a Symbol," do
+      let(:expected_symbol) { "string" }
+
+      it { is_expected_as_block.to raise_error(ArgumentError) }
+    end
+
+    context "when an object equal to an argument," do
+      context "object and argument is only alphabets," do
```

Comment:
Good context! If you could it at all, I'd like you to write tests to check whether a string includes only numbers and a string is an empty string.